backpack/custom_module/graph_utils.py

```diff
@@ -196,11 +237,181 @@ def _transform_flatten_to_module(module: Module, debug: bool) -> GraphModule:
     graph.lint()
 
     if debug:
-        print(f"\tFlatten transformed: {len(nodes)}")
+        print(f"\tFlatten functions transformed: {len(nodes)}")
+
+    return GraphModule(module, graph)
+
+
+def _transform_get_item_to_module(module: Module, debug: bool) -> GraphModule:
+    target = "<built-in function getitem>"
+    if debug:
+        print(f"\tBegin transformation: {target} -> ReduceTuple")
+    graph: Graph = BackpackTracer().trace(module)
+
+    nodes = [
+        n for n in graph.nodes if n.op == "call_function" and target in str(n.target)
+    ]
+    for node in nodes:
+        _change_node_to_module(
+            node,
+            "reduce_tuple",
+            module,
+            ReduceTuple(index=node.args[1]),
+            (node.args[0],),
+        )
+
+    graph.lint()
+    if debug:
+        print(f"\tReduceTuple transformed: {len(nodes)}")
+    return GraphModule(module, graph)
+
+
+def _transform_permute_to_module(module: Module, debug: bool) -> GraphModule:
+    target1 = "permute"
+    target2 = "<built-in method permute"
```

Comment:
missing `>`?

Reply:
No, because full name is longer and object specific.